src/main/jbake/content/docs/components/alerts/index.adoc

```diff
@@ -29,27 +29,6 @@ Other Java 8 compatible runtime environments _should_ work as well.
 Hawkular source code is managed with link:https://maven.apache.org/[Apache Maven]. A minimal version 3.2.x is needed
 to build Hawkular Alerts project.
 
-=== Wildfly 9
-
-http://wildfly.org/downloads/[Download a Wildfly 9.0.1.Final] archive and expand it somewhere on your disk.
-
-=== Apache Cassandra
```

Comment:
Has this part gone on purpose? I see that you still link to apache cassandra, but the removed information could still be useful.
I can imagine though that we put this section on our docs once and then link to it from the various places it is needed.